Revit_Core_Engine/Create/View/Viewport.cs

```diff
@@ -44,6 +44,9 @@ public static Viewport Viewport(this ViewSheet sheet, View view, ElementId viewp
         {
             Document document = sheet.Document;
 
+            if (viewportPlacementPoint == null)
```

Comment:
I would add `RecordNote` here. Also could you please test how does this change affect the behaviour of pushing the viewport via Revit adapter? Something to be added as a test script for this PR

Reply:
I removed this check, if `viewportPlacementPoint` is null, `Viewport` will be placed in default position `(0, 0, 0)`. I added `RecordNote`.